Dockerfile

```diff
@@ -23,15 +23,23 @@ RUN mkdir -p /config/.config/openbox \
     && echo '<?xml version="1.0" encoding="UTF-8"?> \
 <openbox_config> \
   <applications> \
-    <application  type="normal" class="jetbrains-studio"> \
+    <application type="normal" class="jetbrains-studio"> \
       <decor>no</decor> \
       <fullscreen>yes</fullscreen> \
     </application> \
   </applications> \
 </openbox_config>' > /config/.config/openbox/rc.xml
 
 
-CMD ["/init"]
+# Add security settings for opening links
+RUN apk add --no-cache chromium \
```

Comment:
I was wanting to explore if it was possible to have the links fire in the user's browser as opposed to hosting chromium within the container. Haven't looked into if that's possible yet.